core/src/main/java/org/web3j/protocol/rx/JsonRpc2_0Rx.java

```diff
@@ -138,35 +139,21 @@ public JsonRpc2_0Rx(Web3j web3j, ScheduledExecutorService scheduledExecutorServi
     private Flowable<EthBlock> replayBlocksFlowableSync(
             DefaultBlockParameter startBlock,
             DefaultBlockParameter endBlock,
-            boolean fullTransactionObjects,
-            boolean ascending) {
-
-        BigInteger startBlockNumber = null;
-        BigInteger endBlockNumber = null;
+            boolean isFullObjects,
```

Comment:
i don't like this name change. maybe `containsFullTransactionObjects` ?

`isFullObjects` is too abstract here

Reply:
yes, i agree)